[2023-11-02T16:04:50.144+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: es-schemas-creation-dag.create_es_index scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-02T16:04:50.155+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: es-schemas-creation-dag.create_es_index scheduled__2023-11-01T00:00:00+00:00 [queued]>
[2023-11-02T16:04:50.155+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 1
[2023-11-02T16:04:50.172+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): create_es_index> on 2023-11-01 00:00:00+00:00
[2023-11-02T16:04:50.178+0000] {standard_task_runner.py:57} INFO - Started process 354 to run task
[2023-11-02T16:04:50.182+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'es-schemas-creation-dag', 'create_es_index', 'scheduled__2023-11-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/create_es_indeces.py', '--cfg-path', '/tmp/tmp70_uh0uw']
[2023-11-02T16:04:50.185+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask create_es_index
[2023-11-02T16:04:50.418+0000] {task_command.py:415} INFO - Running <TaskInstance: es-schemas-creation-dag.create_es_index scheduled__2023-11-01T00:00:00+00:00 [running]> on host 62132fa558d6
[2023-11-02T16:04:50.744+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='es-schemas-creation-dag' AIRFLOW_CTX_TASK_ID='create_es_index' AIRFLOW_CTX_EXECUTION_DATE='2023-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-11-01T00:00:00+00:00'
[2023-11-02T16:04:50.746+0000] {taskinstance.py:1935} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/create_es_indeces.py", line 24, in create_es_index
    es_conn = es_hook.get_conn()
  File "/usr/local/lib/python3.8/functools.py", line 967, in __get__
    val = self.func(instance)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/elasticsearch/hooks/elasticsearch.py", line 181, in get_conn
    return self._get_elastic_connection()
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/providers/elasticsearch/hooks/elasticsearch.py", line 174, in _get_elastic_connection
    client = Elasticsearch(self.hosts, **self.es_conn_args)
  File "/home/airflow/.local/lib/python3.8/site-packages/elasticsearch/_sync/client/__init__.py", line 331, in __init__
    node_configs = client_node_configs(
  File "/home/airflow/.local/lib/python3.8/site-packages/elasticsearch/_sync/client/utils.py", line 105, in client_node_configs
    node_configs = hosts_to_node_configs(hosts)
  File "/home/airflow/.local/lib/python3.8/site-packages/elasticsearch/_sync/client/utils.py", line 151, in hosts_to_node_configs
    node_configs.append(url_to_node_config(host))
  File "/home/airflow/.local/lib/python3.8/site-packages/elastic_transport/client_utils.py", line 216, in url_to_node_config
    raise ValueError(
ValueError: URL must include a 'scheme', 'host', and 'port' component (ie 'https://localhost:9200')
[2023-11-02T16:04:50.812+0000] {taskinstance.py:1398} INFO - Marking task as FAILED. dag_id=es-schemas-creation-dag, task_id=create_es_index, execution_date=20231101T000000, start_date=20231102T160450, end_date=20231102T160450
[2023-11-02T16:04:50.842+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 4 for task create_es_index (URL must include a 'scheme', 'host', and 'port' component (ie 'https://localhost:9200'); 354)
[2023-11-02T16:04:50.875+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2023-11-02T16:04:50.966+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
